docs/configuration.md

```diff
@@ -0,0 +1,214 @@
+# Configuration variables
+
+Arctic has several tuning knobs under [arctic/_config.py](https://github.com/manahl/arctic/blob/master/arctic/_config.py) which affect the functionality of certain modules.
+
+Most of these variables are initialized via environment variables, which are explained in the rest of this section.
+
+
+<br><br>
+
+## VersionStore
+
+### STRICT_WRITE_HANDLER_MATCH
+
+Controls if Arctic can only match the intended handlers for the data type. If set to true it prevents falling back to pickling if the matching handler (based on the type of data) can't serialize without objects.
+
+```
+export STRICT_WRITE_HANDLER_MATCH=1
+```
+
+
+
+<br><br>
+
+
+## NdArrayStore
+
+### CHECK_CORRUPTION_ON_APPEND
+
+Enables more thorough sanity checks for detecting data corruption when issuing appends. The checks will introduce a 5-7% performance hit. This is disabled by default.
```

Comment:
Is there a programmatic way to do this check? If not, should there be?

Reply:
There are already two implementations, well documented:
https://github.com/manahl/arctic/blob/master/arctic/store/_version_store_utils.py#L321
and
https://github.com/manahl/arctic/blob/master/arctic/store/_version_store_utils.py#L344

I will have to slightly update this for the FW pointers = ENABLED case were we no longer have parents.